scripts/branch_rename_migration.py

```
@@ -0,0 +1,251 @@
+        if parts[0] == 'feat':
+            return f"feature/{'/'.join(parts[1:])}"
+        elif parts[0] == 'fix':
+            return f"bugfix/{'/'.join(parts[1:])}"
+        elif parts[0] == 'doc':
+            return f"docs/{'/'.join(parts[1:])}"
+        # If already has correct prefix, just return None
+        elif parts[0] in ['feature', 'bugfix', 'hotfix', 'refactor', 'docs']:
+            return None
+        # Handle fix-* branches that should be bugfix/*
+        elif parts[0] == 'fix' or 'fix' in parts[0]:
+            return f"bugfix/{'/'.join(parts[1:]) if len(parts) > 1 else parts[0]}"
```

Pylint (4.0.2)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)